1080i/MyPVRRecordings.xml

```diff
@@ -173,12 +173,20 @@
 						<font>font36_title</font>
 						<label>$INFO[ListItem.Title]</label>
 					</control>
+					<control type="label">
+						<top>240</top>
+						<width>560</width>
+						<height>262</height>
+						<label>$INFO[ListItem.EpisodeName]</label>
+						<font>font14</font>
+					</control>
 					<control type="textbox">
-						<top>254</top>
+						<top>282</top>
 						<width>750</width>
 						<height>526</height>
 						<align>justify</align>
 						<label>$INFO[ListItem.Plot]</label>
+						<textcolor>grey</textcolor>
```

Comment:
what about makin EpisodeName (or whatever is shown above plot) italic? That way we would have a consistent look across different sections, we never use grey font for plot anywhere.